hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableOutputFormat.java

```diff
@@ -345,14 +346,18 @@ public static CarbonLoadModel getLoadModel(Configuration conf) throws IOExceptio
                     CarbonCommonConstants.LOAD_BATCH_SORT_SIZE_INMB,
                     CarbonCommonConstants.LOAD_BATCH_SORT_SIZE_INMB_DEFAULT))));
 
-    model.setBadRecordsLocation(
-        conf.get(BAD_RECORD_PATH,
-            carbonProperty.getProperty(
-                CarbonLoadOptionConstants.CARBON_OPTIONS_BAD_RECORD_PATH,
-                carbonProperty.getProperty(
-                    CarbonCommonConstants.CARBON_BADRECORDS_LOC,
-                    CarbonCommonConstants.CARBON_BADRECORDS_LOC_DEFAULT_VAL))));
-
+    String badRecordsPath = conf.get(BAD_RECORD_PATH);
+    if (StringUtils.isEmpty(badRecordsPath)) {
+      badRecordsPath =
```

Comment:
carbon table properties should be checked first then take from configuration

Reply:
conf.get will give bad_record_path from load options